temporal-test-server/src/main/java/io/temporal/internal/testservice/TestWorkflowMutableStateImpl.java

```diff
@@ -712,20 +711,28 @@ private void processScheduleNexusOperation(
       long workflowTaskCompletedId) {
     attr = validateScheduleNexusOperation(attr);
     Endpoint endpoint = nexusEndpointStore.getEndpointByName(attr.getEndpoint());
-    StateMachine<StateMachines.NexusOperationData> operationStateMachine =
-        newNexusOperation(endpoint);
+    StateMachine<StateMachines.NexusOperationData> operation = newNexusOperation(endpoint);
     long scheduleEventId = ctx.getNextEventId();
-    nexusOperations.put(scheduleEventId, operationStateMachine);
+    nexusOperations.put(scheduleEventId, operation);
 
-    operationStateMachine.action(Action.INITIATE, ctx, attr, workflowTaskCompletedId);
+    operation.action(Action.INITIATE, ctx, attr, workflowTaskCompletedId);
+    // Use TIMEOUT_TYPE_SCHEDULE_TO_START for timing out an individual start request
+    ctx.addTimer(
+        ProtobufTimeUtils.toJavaDuration(operation.getData().taskRetryPolicy.getInitialInterval()),
```

Comment:
The attempt timeout isn't related to the retry policy. In the real server we default to 10 seconds.

Reply:
I set the default to 5 seconds in the test server, mostly just to make the tests run a bit faster. If we think that's a bad experience, I am happy to switch it back to 10s.